### PR TITLE
fix some orthographic errors handle ablative cases around numbers

### PR DESCRIPTION
**Pull request by marmeladze**
https://github.com/svenfuchs/rails-i18n/pull/779#discussion_r204871949

### CODE REVIEW

File: rails/locale/az.yml [hunk 1]
```diff
@@ -109,29 +109,29 @@ az:
       accepted: qəbul olunmalıdır
       blank: boş ola bilməz
       present: boş olmalıdır
-      confirmation: təsdiqə uygun deyil
+      confirmation: təsdiqə uyğun deyil
       empty: boş ola bilməz
-      equal_to: "%{count}-ə bərabər olmalıdır"
+      equal_to: "%{count} simvoldan ibarət olmalıdır"
```
[the remark this answers is not in the document]
Comment (marmeladze) [reply rails/locale/az.yml]: sorry, it was a total confusion. i was meant to translate only length messages - were not able to find proper expressions for numericality validations.

 * 1-ə bərəbər olmalıdır / equal_to 1
 * 6-dan böyük olmalıdır / greater_than 6
 * 7-dən kiçik olmalıdır  / less_than 7
 * 9-dan kiçik olmamalıdır / greater_than_or_equal_to 9
 * 11-dən böyük olmamalıdır / less_than_or_equal_to 11

i've just reverted them back and kept others.

File: rails/locale/az.yml [hunk 2]
```diff
@@ -109,29 +109,29 @@ az:
...
-      confirmation: təsdiqə uygun deyil
+      confirmation: təsdiqə uyğun deyil
       empty: boş ola bilməz
       equal_to: "%{count}-ə bərabər olmalıdır"
       even: cüt olmalıdır
       exclusion: qorunur
       greater_than: "%{count}-dən böyük olmalıdır"
-      greater_than_or_equal_to: böyük və ya %{count}-ə bərabər olmalıdır
-      inclusion: siyahiyə daxil deyil
+      greater_than_or_equal_to: "%{count}-dən az olmamalıdır"
```
Comment (NARKOZ) [inline rails/locale/az.yml]: meaning of the message was changed by dropping "equal to"

File: rails/locale/az.yml [hunk 3]
```diff
@@ -109,29 +109,29 @@ az:
...
+      inclusion: siyahıya daxil deyil
       invalid: yanlışdır
       less_than: "%{count}-dən kiçik olmalıdır"
-      less_than_or_equal_to: kiçik və ya %{count}-ə bərabər olmalıdır
+      less_than_or_equal_to: "%{count}-dən çox olmamalıdır"
```
Comment (NARKOZ) [inline rails/locale/az.yml]: meaning of the message was changed by dropping "equal to"

File: rails/locale/az.yml (hunk 2 above)
Comment (marmeladze) [reply rails/locale/az.yml]: _greater than or equal to n_ means n is our minimum acceptable value. expressing such way (n-dən az olmamalıdır) in azerbaijani, i guess is more clear.

Comment (NARKOZ) [reply rails/locale/az.yml]: That way it is not clear if n is included or excluded. The message should be explicit. See Turkish for example.

**Comment by NARKOZ** [comment]
@marmeladze are you going to fix the changes?

**Pull request closed without merging**